### DISCUSSION TITLE
Update tutoriais for latest (1.3.6) version

### DISCUSSION THREAD

**Discussion by gforti84** [Eomys/pyleecan#473] [General]
When are the tutorials going to be updated or where can I find which version are they suited for?

Thanks for the help.

**Comment by BonneelP** [comment]
Hello, 
Sorry for the long time without answer, I was in holiday. 
On Friday I have updated several tutorials and release pyleecan 1.3.7 (cf 9176028ac23ec60745e8fa37bfde5b6bf5336ec3). I still have some tutorials and the website to update but this is ongoing. The first cell of the tutorial print the version of pyleecan when it was run for the last time. 

Best regards,
Pierre
